src/DocBlock/Annotation.php

```diff
@@ -157,7 +157,7 @@ public function getTypes()
      */
     public function setTypes(array $types)
     {
-        $pattern = '/'.preg_quote($this->getTypesContent()).'/';
+        $pattern = '/'.preg_quote($this->getTypesContent(), '/').'/';
```

Comment:
The reasoning is the fact that preg_quote is not escaping '/' by default and here used exactly this separator. I didn't check deeper than at this level.